newsletters/011.md

```diff
@@ -18,7 +18,7 @@ Most of the code is already open-source and can be obtained from repositories su
 
 + Linux Official Community Upstream work:
 
-  + 
+  + [[PATCH v4 00/10] Add Milk-V Pioneer RISC-V board support](https://lore.kernel.org/linux-riscv/cover.1696433229.git.unicorn_wang@outlook.com/): the minimal system submitted version 4 patches.
```

Comment:
Seems like this translation does not match its original meaning?

Reply:
Changed and re-submitted, please check again.